### PR TITLE
chore: remove polyfills

### PR DESCRIPTION
**Pull request by tjuanitas**

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -33,9 +33,9 @@ module.exports = {
     plugins: [
         '@babel/plugin-syntax-dynamic-import',
         '@babel/plugin-transform-flow-strip-types',
+        '@babel/plugin-transform-class-properties', // Order matters: `transform-class-properties` must come after `transform-flow-strip-types`
```
Comment (jpan-box) [inline babel.config.js:36]: What happens if this is not in order?

**Review by jpan-box — DISMISSED** [review]

**Review by greg-in-a-box — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -249,9 +248,7 @@
         "mousetrap": "^1.6.3",
         "msw": "^2.1.5",
         "msw-storybook-addon": "^2.0.2",
-        "mutationobserver-shim": "^0.3.3",
-        "npm-run-all": "^4.1.3",
-        "nsp": "^3.2.1",
```
Comment (tjuanitas) [inline package.json:254]: package is deprecated and service is down. i dont see where it's used?
https://www.npmjs.com/package/nsp

File: package.json [hunk 3]
```diff
@@ -249,9 +248,7 @@
         "mousetrap": "^1.6.3",
         "msw": "^2.1.5",
         "msw-storybook-addon": "^2.0.2",
-        "mutationobserver-shim": "^0.3.3",
```
Comment (tjuanitas) [inline package.json:252]: mutationobserver is widely available

File: package.json [hunk 4]
```diff
@@ -199,7 +199,6 @@
         "conventional-changelog-cli": "^2.0.31",
         "conventional-github-releaser": "^3.1.3",
         "copyfiles": "^2.2.0",
-        "core-js": "^3.6.4",
```
Comment (tjuanitas) [inline package.json:202]: only see two direct uses of this - map and set which are widely supported

File: scripts/jest/enzyme-adapter.js [hunk 5]
```diff
@@ -1,8 +1,3 @@
-/* eslint-disable import/no-extraneous-dependencies */
-import 'core-js/es/map';
-import 'core-js/es/set';
-import 'raf/polyfill';
```
Comment (tjuanitas) [inline scripts/jest/enzyme-adapter.js:4]: requestAnimationFrame has wide support

File: package.json [hunk 6]
```diff
@@ -282,7 +278,7 @@
         "react-styleguidist": "^13.1.3",
         "react-tether": "^1.0.5",
         "react-textarea-autosize": "^8.5.3",
-        "regenerator-runtime": "^0.13.3",
```
Comment (tjuanitas) [inline package.json:285]: don't know if we technically still need this but not confident enough to remove yet

File: babel.config.js (hunk 1 above)
Comment (tjuanitas) [reply babel.config.js]: was throwing a random error in one of the components. how i understand the issue is that babel was transpiling code in an order which conflicted with code that hadn't been transpiled

File: package.json (hunk 6 above)
Comment (greg-in-a-box) [reply package.json:285]: wasn't this one of those things that isnt referenced but is used ?

Comment (tjuanitas) [reply package.json:285]: don't know but it's directly imported in the elements: https://github.com/box/box-ui-elements/blob/master/src/elements/content-explorer/ContentExplorer.js#L7

Comment (greg-in-a-box) [reply package.json:285]: im pretty sure i tried to remove this one time but i cant remember what broke when i took it out

**Review by greg-in-a-box — APPROVED** [review]

**Review by jpan-box — APPROVED** [review]

**Pull request merged**